subsys/metrics/src/main/conf/conf.d/metrics.conf

```diff
@@ -1,10 +1,21 @@
 [metrics]
 enabled = false
-reporter = graphite.reporter
+reporter = graphite.reporter;zabbix.reporter
```

Comment:
Please make this a comma-separated list, not semicolon-separated. It's more conventional for other configurations you often see.

Regarding documentation for this file (and default-metrics.conf):

**Need to remove internal URLs from these files.** Just put in a placeholder example URL instead. I'd leave the URL path intact to prompt the user.

Also, FWIW, we should set `enabled = false` for the metrics config file (both the default and the one in src/main/conf. 

And, we should comment each section, like this:

```
    ############################################
    # GraphiteDB reporter options
    ############################################
    #
    # graphite.reporter.hostname = localhost
    # graphite.reporter.port = 2003
    # graphite.reporter.prefix = local.one
    # graphite.reporter.simple.period = 30
    # graphite.reporter.jvm.period = 300
    # graphite.reporter.healthcheck.period = 600
```

We should use the following comment to show how to enable different reporters:

```
    ######################################
    # List of enabled reporters (comma-separated)
    ######################################
    #
    # reporters = graphite, zabbix
```

I think you should change `reporter` to `reporters` for the main configuration options like `reporter =`. Then, change the `graphite.reporter.*` to just `graphite.*` (and same for zabbix).

Finally, what does `reporter.enable` control? Is this the same as `enabled`? If we still need it, we should put in some documentation about what it's used for.